.github/workflows/run.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: My GitHub Status
-        uses: yihong0618/my_github@v1
+        uses: ./
```

Comment:
This is why I choose to change it : https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-docker-container-action#example-using-a-private-action . 

So it will be easier for developers and no influence for users for its already published

Reply:
It is fine, we can keep it using `.` for this repository

Reply:
Yes, I will use ./, you are right.